cartridges/int_newslettersubscription/cartridge/static/default/css/newsletterForm.css

```diff
@@ -0,0 +1,29 @@
+main {
+  text-align: center;
+}
+
```

Comment:
Static files should not be part of the repo, please add the whole static folders for each cartridge in gitignore. First  you have to delete the folder all together, add the git ignore change and commit your changes.

Reply:
fixed